charts/pg-db/README.md

```diff
@@ -1,135 +1,76 @@
 | `bucket.s3ca`                     | Put custom CA certificate for your S3-compatible storage                                        |``|
-| `pgPrimary.image`                     | Set this variable if you need to use a custom PostgreSQL image                                        | `percona/percona-postresql-operator:1.4.0-ppg14-postgres-ha`                              |
-| `pgPrimary.resources.requests.memory`                     | Container resource request for RAM                                        | `128Mi`                              |
-| `pgPrimary.tolerations`                     | PostgreSQL container deployment tolerations | `[]`                              |
-| `pgPrimary.volumeSpec.size`                     | PostgreSQL container PVC size | `1G`                              |
-| `pgPrimary.volumeSpec.accessmode`                     | PostgreSQL container PVC accessmode | `ReadWriteOnce`                              |
-| `pgPrimary.volumeSpec.storagetype`                     | PostgreSQL container PVC storagetype | `dynamic`                              |
-| `pgPrimary.volumeSpec.storageclass`                     | PostgreSQL container PVC storageclass | `standard`                              |
-| `pgPrimary.expose.serviceType`                     | K8S service type for the PostgreSQL deployment | `ClusterIP`                              |
-| `pgPrimary.expose.loadBalancerSourceRanges`                     | Allow specific IP source ranges for accessing PostgreSQL | `[]`                              |
-| `pgPrimary.expose.loadBalancerIP` | Static IP address for Primary PostgreSQL service | `` |
-| `pgPrimary.affinity.antiAffinityType`                     | Pod antiAffinity for PostreSQL db pods `preferred/required/disabled` | `preferred`                              |
-| `pgPrimary.affinity.nodeAffinityType`                     | Node antiAffinity for PostreSQL db pods `preferred/required/disabled` | `preferred`                              |
-| `pgPrimary.affinity.nodeLabel`                     | Key/value map with node lables specified for nodeAffinity | `key/value`                              |
-| `pgPrimary.affinity.advanced`                     | User defined affinity structure to be passed to pods without any processing | `key/value`                              |
 | `pmm.enabled` | Enable integration with [Percona Monitoring and Management software](https://www.percona.com/blog/2020/07/23/using-percona-kubernetes-operators-with-percona-monitoring-and-management/) | `false` |
-| `pmm.image.repository`              | PMM Container image repository                                           | `percona/pmm-client` |
-| `pmm.image.tag`                     | PMM Container image tag                                       | `2.37.0`                              |
+| `pmm.image`              | PMM Container image repository                                           | `percona/pmm-client:2.37.0` |
```

Comment:
aren't we missing a lot of new fields like whole `instances` stuff

Reply:
it good question - should we put all them in README?

Reply:
yes, we need to document each field which can be changed in the values